xwiki-platform-core/xwiki-platform-resource/xwiki-platform-resource-api/src/main/java/org/xwiki/resource/ResourceType.java

```diff
@@ -82,4 +94,12 @@ public boolean equals(Object object)
             .append(getId(), rhs.getId())
             .isEquals();
     }
+
+    /**
+     * @return true if this type represents a static resource (i.e: a ResourcesResource or a SkinResource only)
```

Comment:
That's not the best definition  IMO :) A static resource contains static content and must be served by the Servlet Container directly, without going through XWiki's processing and handling.